src/datastructures.apib

```diff
@@ -140,3 +140,121 @@
         + timeTracking.deleted
         + timeTracking.updated
         + user.deactivated
+
+## WorkOrderData (object)
++ customer (object, required)
+    + type (enum[string], required)
+        + Members
+            + contact
+            + company
+    + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
++ datetime: `2017-05-26T10:01:49+00:00` (string, required)
```

Comment:
Q : what does this `datetime` represent?

Reply:
It's the date when the work order was performed. I've renamed it to `performed_at`. We will check the naming with product anyway.